src/System.ValueTuple/src/System.ValueTuple.csproj

```diff
@@ -40,6 +40,12 @@
     </When>
   </Choose>
 
+  <!-- Also put into build folder for packages.config support. -->
+  <ItemGroup>
+    <Content Include="buildTransitive\net471\System.ValueTuple.targets" PackagePath="buildTransitive\net471\$(PackageId).targets" />
+    <Content Include="buildTransitive\net471\System.ValueTuple.targets" PackagePath="build\net471\$(PackageId).targets" />
+  </ItemGroup>
+
```

Comment:
The packages.config case is more convoluted.  The only way RAR saw a redirect was needed here was if I referenced two different libraries with different VT versions.  Then it would produce a redirect *and* pull in the dll from the project directory of the referenced project.  I think it's better to not redirect when it's inbox (even if RAR is copying the assembly), so we should keep this in.

I was able to test that applying the new version makes the redirect go away.

Reply:
Also confirmed this fixes the repro case I added to the issue.